### PR TITLE
LPS-145325 Remove FF item selector single uploader

### PR DESCRIPTION
**Pull request by boton**
**Issue:** https://issues.liferay.com/browse/LPS-145325

I have not been able to reduce the logic as much as I would have liked because the old uploader is still in use on [`item-selector-upload-web`](https://github.com/liferay/liferay-portal/blob/30d870a990b049b4b2bb90249b8e303868292c17/modules/apps/item-selector/item-selector-upload-web/src/main/resources/META-INF/resources/js/index.es.js#L16).

I prefer to [create a new task](https://issues.liferay.com/browse/LPS-160919) to apply the new ItemSelectorSingleFileUploaderEnabled on [`item-selector-upload-web`](https://github.com/liferay/liferay-portal/tree/30d870a990b049b4b2bb90249b8e303868292c17/modules/apps/item-selector/item-selector-upload-web) I already have the logic clean on boton#231, I will apply it after update the `item-selector-upload-web`.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

File: modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp [hunk 1]
```diff
@@ -647,7 +623,7 @@ SearchContainer<?> searchContainer = new SearchContainer(renderRequest, itemSele
 			editImageURL: '<%= editImageURL.toString() %>',
 		</c:if>
 
-		ffItemSelectorSingleFileUploaderEnabled: <%= FFItemSelectorSingleFileUploaderConfigurationUtil.enabled() %>,
+		itemSelectorSingleFileUploaderEnabled: <%= true %>,
```
Comment (AliciaGarciaGarcia) [inline modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp:626]: Can this variable change?
if not. it's really needed?

Comment (boton) [reply modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp:626]: > Can this variable change?

yes, unfortunately on [`item-selector-upload-web`](https://github.com/liferay/liferay-portal/blob/30d870a990b049b4b2bb90249b8e303868292c17/modules/apps/item-selector/item-selector-upload-web/src/main/resources/META-INF/resources/upload.jsp#L63-L81) isn't passed and the itemSelectorSingleFileUploaderEnabled and it is disabled.

I hope isn't needed after [LPS-160919](https://issues.liferay.com/browse/LPS-160919) is done in near future.

**Comment by boton** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
ci:stop

**Comment by austinchiang** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:reevaluate:1377401_4333

**Comment by boton** [comment]
The failure is related but the markup is updated this label doesn't exist now. 

```
java.lang.Exception: Element is not present at "//div[@id='itemSelectorUploadContainer']//label[contains(.,'Select File')] | //div[contains(@id,'ItemSelectorContainer')]//label[contains(.,'Select File')]"
```

We need to update the test :D

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
cc @liuxingye @Jeremy-Ch could you please helps us with the poshi failure? I think we need an update in the poshi test. Thanks in advance!

**Comment by Jeremy-Ch** [comment]
See the new pull request: https://github.com/liferay-lima/liferay-portal/pull/3396

**Pull request closed without merging**